docs/data-dictionary/types/saneringskrediet_voorstand.yml

```diff
@@ -0,0 +1,6 @@
+type: number
```

Comment:
@Frank-SBN Is voorstand en achterstand rente gegeven of een verschil tussen vorderingen en betalingen?

Reply:
@Frank-SBN Krediet berekent het proces andersom. Theoretisch saldo wordt berekend op basis van kredietsom, betalingen, voorstand en achterstand.